src/main/scala/authentikat/jwt/JsonWebSignature.scala

```diff
@@ -52,7 +52,7 @@ object JsonWebSignature {
 
   abstract class Algorithm
 
-  case object none extends Algorithm
+  case object NONE extends Algorithm
```

Comment:
Good catch - probably was named none as it's an object instances,  but I think this should actually be "None" - the Scala standard for Object names and Constants is actually Camel case. Full Caps is Java constant naming convention. http://docs.scala-lang.org/style/naming-conventions.html